packages/client/components/PokerSidebarEstimateSection.tsx

```diff
@@ -84,13 +84,11 @@ const PokerSidebarEstimateSection = (props: Props) => {
               <ScrollWrapper ref={provided.innerRef}>
                 {stages!.map((stage, idx) => {
                   const {task, issue} = stage
-                  const content = task?.content || issue?.summary
+                  const content = task?.plaintextContent.split('\n')[0] || issue?.summary
```

Comment:
i thought we should only show the first block of any tasks with potential multiple blocks